src/components/navigation-logo/navgation-logo.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-navigation-logo",
+  styleUrl: "navigation-logo.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavigationLogo {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavigationLogoElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the URL destination of the component, which can be set as an absolute or relative path.*/
+  @Prop({ reflect: true }) href: string;
+
+  /** Describes the appearance or function of the `thumbnail`. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the subtext to display, such as an organization or application description. */
+  @Prop() subtext: string;
+
+  /** Specifies the text to display, such as a product name.*/
+  @Prop() text: string;
+
+  /** When `true`, displays the `text` and `subText` contents. */
+  @Prop({ reflect: true }) textEnabled = false;
+
+  /** Specifies the `src` to an image. */
+  @Prop() thumbnail: string;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  render(): VNode {
+    return (
+      <Host>
+        <a href={this.href} tabindex={0}>
```

Comment:
the tabIndex still needs to be set here because the default 0 is applied only when href has a valid value or an empty string. I tried adding empty string which results in reload of page. Later changed to "#" which will a11y instructions.

Reply:
Maybe you could do `tabindex={!this.href ? 0 : null}`. href should be a required prop though right?

Reply:
`tabindex={!this.href ? 0 : null}` this will over-ride the default tabindex value `0` with `null` when href is defined and `href` is not a required prop.

Reply:
Should we add `rel` & `target` prop here?

Reply:
maybe `href` should just be required?

Reply:
I think there are times where clicking the logo might just emit an event to reset a route or something it might not always be a href to navigate.

Reply:
If that's the case, it should probably follow `calcite-button` logic and render either a button or a anchor depending on the existence of href.

Reply:
i don't think `href` should be a required attribute for logo. Is there an added advantage of rendering a button when href is `undefined` besides click event being emitted on `Space` keydown ?

Reply:
I think it would just be a11y concerns. An anchor should take you somewhere (either on the page or off) whereas as button should perform some action.

Reply:
when href is `undefined` & text-enabled is `false` , VoiceOver is reads out the logo as ` {label} name, image`.
when href is `undefined` & text-enabled is `true` ,  VoiceOver focuses the image reading the instruction as ` {label} name, image`  and then user can move on to the text which is read out.
 
 The only time VoiceOver recognizes logo as `link` is when user adds href.  When both `href` and `label` are `undefined` , the instructions recognizes logo as `clickable` element.
 
 Tested with VoicOver and safari 16.4

Reply:
synced-up with kitty today. 

Updates regarding navigation-logo a11y:

- tabindex default value is removed  because logo shouldn't be in the focus order when href is null value
- AT user should be able to hear out image  (if label is defined), heading & description context 
- Screen readers will recognize logo as link if href is defined.